subworkflows/nf-core/fastq_align_star/main.nf

```diff
@@ -1,5 +1,8 @@
-include { STAR_ALIGN              } from '../../../modules/nf-core/star/align/main'
-include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+include { STAR_ALIGN                                                       } from '../../../modules/nf-core/star/align/main'
+include { RSEM_PREPAREREFERENCE as MAKE_TRANSCRIPTS_FASTA                  } from '../../../modules/nf-core/rsem/preparereference'
```

Comment:
why using an alias?

Reply:
Ported over from rnaseq. But don't worry- it's about to go :-)